Foreground.skin.php

```diff
@@ -136,7 +136,7 @@ public function execute() {
 					<ul id="toolbox-dropdown" class="dropdown">
 						<?php foreach ( $this->getToolbox() as $key => $item ) { echo $this->makeListItem($key, $item); } ?>
 						<?php if ($wgForegroundFeatures['showRecentChangesUnderTools']): ?><li id="n-recentchanges"><?php echo Linker::specialLink('Recentchanges') ?></li><?php endif; ?>
-						<?php if ($wgForegroundFeatures['showHelpUnderTools']): ?><li id="n-help" <?php echo Linker::tooltip('help') ?>><a href="/wiki/Help:Contents"><?php echo wfMessage( 'help' )->text() ?></a></li><?php endif; ?>
+						<?php if ($wgForegroundFeatures['showHelpUnderTools']): ?><li id="n-help" <?php echo Linker::tooltip('help') ?>><a href="<?php echo Skin::makeInternalOrExternalUrl( wfMessage( 'helppage' )->inContentLanguage()->text() )?>"><?php echo wfMessage( 'help' )->text() ?></a></li><?php endif; ?>
```

Comment:
Because message units don't work that way on a localised MW install. Whatever the default language of the wiki is the ruling message unit and all others must work off of them. When you think localised URL's think like a Highlander- "there can only be one" especially when it comes to URLs for a page. inContentLanguage will make sure the URL will output the correct URL in the default content language of the wiki.

Your method of adding message units to the .json file will create multiple URLs for the main help page.

```
Help:Contents  // English version
Aide:Contenu  // French version
Ajuda:Continguts // Catalan version
```
If someone has the Translate extension installed and they want to link to the correct language of a user then they would set MediaWiki:Helppage to `Special:MyLanguage/Help:Contents` for example if they had an `en` (English) wiki. If it is a dedicated French language wiki, they would already see Aide as the 'Help' namespace so setting their MediaWiki:Helppage to Aide:Contenu would take their users to `http://theirdomain.org/wiki/Adie:Contenu`

Remember, namespaces are value set in the db. Help is 12 and the output is just the set value based on the localised language.